ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-ref.yaml

```diff
@@ -15,9 +15,17 @@ ref:
     documentation: |-
       File in which we will parse to find uuid, start and end time
   - name: PODS_PER_NODE
-    default: "200"
+    default: "100"
```

Comment:
do we need to lower this?

Reply:
@paigerube14 this was to mimic what we are doing with node-density heavy --- Last I heard we wanted those things to mimic each other. https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml#L21